roxie/udplib/udptrr.cpp

```diff
@@ -1200,6 +1200,8 @@ class CReceiveManager : implements IReceiveManager, public CInterface
         #endif
             UdpRequestToSendMsg msg;
             unsigned timeout = 5000;
+            unsigned lastReport = 0;
```

Comment:
Should we init this to msTick() ?  But certainly no big deal.

Reply:
No, I want the first time to come out straight away rather than be delayed for a minute